app/dashboards/dashboard_manifest.rb

```diff
@@ -11,6 +11,7 @@ class DashboardManifest
   # to work correctly.
   DASHBOARDS = [
     :locations,
+    :jobs
     :events,
   ]
 
```

Comment:
Hey @anonyo, I assume this is working, but this doesn't look like a valid array, missing a comma after `jobs`.